EpikCord/client/websocket_client.py

```diff
@@ -361,8 +361,7 @@ async def _guild_create(self, data: discord_typings.GuildCreateEvent):
         for thread in data["threads"]:
             self.channels.add_to_cache(data["id"], Thread(self, thread))
 
-        return guild
-
+        await self.dispatch("guild_create", guild)
```

Comment:
Can you explain this change, not sure of what happening/being updated here

Reply:
So the old event handling system would call the internal function, and then it would call the user defined event with the returned items of the internal event, and I've changed that so that it is explicit by using `dispatch`.